src/shared/analytics/analytics.background.ts

```diff
@@ -327,6 +328,21 @@ function trackAppEvents({ account }: { account: Account }) {
     }
   });
 
+  emitter.on('finalQuoteReceived', ({ quote, formView, scope }) => {
+    const params = createParams({
+      request_name: 'swap_form_filled_out',
+      screen_name: scope === 'Swap' ? 'Swap' : 'Bridge',
```

Comment:
I think, we can do 
`invariant(scope === 'Swap' || scope === 'Bridge')`
And then
```
screen_name: scope,
client_scope: scope,
```